packages/tables/resources/views/components/actions/cell.blade.php

```diff
@@ -1,7 +1,7 @@
 <x-filament-tables::cell
     :attributes="\Filament\Support\prepare_inherited_attributes($attributes)"
 >
-    <div class="whitespace-nowrap px-3 py-4">
+    <div class="fi-ta-actions-cell whitespace-nowrap px-3 py-4">
```

Comment:
In that case, it would not solve the ease of selecting this element. If I move the class up, and I want to change the padding on `<div class="whitespace-nowrap px-3 py-4">` I will have to use selectors like `> div` or target `.px-3{ }` in order to override the padding. What would you recommend?

Reply:
We'll probably have to add a class to the `<div>` too eventually.
`fi-ta-actions-cell` should still be on the root element in my opinion, to be consistent. Then a new class can be added to the `<div>`.